docs/dev/vagrant.rst

```diff
@@ -105,26 +94,19 @@ Once you have everything set up, normal usage of your vagrant dev server should
 Before you start anything: ::
 
     vagrant up
-    fab open_db
 
 To run your dev server: ::
 
-    fab run_devserver
+    fab runserver
 
 Other useful command examples: ::
 
     fab manage:shell_plus
-    fab manage:'migrate program'
```

Comment:
Sure, that's a good one. I probably wouldn't run it on my dev server in real life, but I won't complain about more people knowing it exists, and it points out the existence of the `psql